hotline/notifications/forms.py

```diff
@@ -7,3 +7,17 @@ class UserNotificationQueryForm(forms.ModelForm):
     class Meta:
         model = UserNotificationQuery
         fields = ['name']
+
+
+class UserSubscriptionDeleteForm(forms.Form):
+    subscriptions = forms.ModelMultipleChoiceField(required=True, widget=forms.CheckboxSelectMultiple, queryset=None)
+
+    def __init__(self, *args, **kwargs):
+        self.user = kwargs.pop('user')
```

Comment:
In Python 2, we had to do that `kwargs.pop` business, but in 3, we can do things the more readable way:
```
def __init__(self, *args, user, **kwargs):
    self.user = user
```